roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml

```diff
@@ -45,8 +45,29 @@
     body:
       type: "alertmanager"
       name: "{{ cluster_name }}"
+      default_route:
+        slack:
+          channel_id: "{{ slack_id }}"
+          enabled: true
```

Comment:
where is this enabled: true coming from? is this part of the request to be sent to Grafana OnCall? Or is it part of our CR? could you explain this a bit?

Reply:
yes it is required for the body to be posted towards grafana on call when slackId is added, this is the format example

curl "https://oncall-prod-eu-west-0.grafana.net/oncall/api/v1/integrations/"   --request POST   --header "Authorization:"   --header "Content-Type: application/json"   --data '{
      "type":"alertmanager",
      "name":"tno4-ams-test",
      "default_route": {
        "slack": {
          "channel_id": "C06G60ZD4JZ",
          "enabled": true
        }
    }
}'